Dockerfile_baseline

```diff
@@ -48,6 +33,9 @@ RUN $PYTHON -m pip install --no-cache mpi4py
 # Install JupyterLab
 RUN $PYTHON -m pip install --no-cache jupyter notebook
 
+# Install PaperMill
```

Comment:
Would it make more sense to be in the same RUN statement where jupyter is installed?

Reply:
From the practical point of view it won't change much, but it creates a logical division of what's installed. From the performance point of view, I don't think it's going to be much faster having the two installations on a single RUN rather than on two. It might create a slightly larger overhead in terms of space, because Docker creates a new layer every RUN. However, the `--no-cache` option should mitigate it.